docs/en/stack/ml/dataframeanalytics.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[[ml-dfanalytics]]
+=== {dfanalytics-cap}
+
+{dfanalytics-cap} allows you to perform different analyses of your data and 
+annotate it with the results. Essentially, as part of its output, {dfanalytics} 
+appends the results of the analysis to the source data. By doing this, it 
+provides additional insights on the data. The process leaves the source index 
+intact, it creates a new index that contains the annotated data. You can slice 
+and dice the data extended with the results as you normally do with any other 
+data set.
+
+IMPORTANT: Using {dfanalytics} assumes you have already done some preprocessing 
```

Comment:
We do not assume that data frame transforms have been used. Please change to something along the lines of ...

`Using {dfanalytics} requires source data to be structured as a two dimensional "tabular" data structure i.e. a data frame. Data frame transforms allow you to easily create data frames which can be used as the source for data frame analytics.`